source/core/src/main/com/csse3200/game/areas/terrain/TerrainChunk.java

```diff
@@ -55,10 +55,10 @@ public void generateTiles(GridPoint2 chunkPos, Map<GridPoint2, TerrainChunk> loa
     // INFO: The Map is equally divied into three areaas. Each area is 16x10 tiles wide.
     inArea = checkAreaType(position);
     switch (inArea) {
-      case MapType.FOREST -> totalTiles = TerrainResource.getTileSize(TileLocation.FOREST);
-      case MapType.WATER -> totalTiles = TerrainResource.getTileSize(TileLocation.WATER);
-      case MapType.FOG -> totalTiles = TerrainResource.getTileSize(TileLocation.FOG);
-      default -> totalTiles = TerrainResource.getTileSize(TileLocation.AIR);
+      case MapType.FOREST -> totalTiles = terrainResource.getForestSize();
```

Comment:
I'm not sure this change is an improvement - it requires a function for each map type which is pointless considering they do the same thing.